src/components/Form/Form.jsx

```diff
@@ -15,12 +16,37 @@ export const Form = () => {
     lastDate: null,
   });
 
+  const [value] = useCollection(getAll());
+
+  const isItemDuplicated = (name) => {
+    const regex = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/g;
```

Comment:
Whoa! now that's what I call a regex! 

Personally, I'm horrible are reading and writing regex and I'm pretty impressed by this. I wonder if giving it a more meaningful name like `punctuation` would be helpful.

Also, I'm wondering if maybe in our input fields where a use inputs the names of the items, we should also restrict their inputs so that they can't add punctuation into the database. That's a design choice though, so you can totally decide how you'd like it. Totally up to you. Maybe users want to be able to put `Beer!` on their list rather than boring old `Beer`.  If you did decide to restrict input values, take a look at my comment about controlled inputs on the other PR.